docs/automated-workflows/dagman-interaction.rst

```diff
@@ -35,12 +34,11 @@ the same time. It will fail in an unpredictable manner, as each instance
 of this same DAG will attempt to use the same file to enforce
 dependencies.
 
-To increase robustness and guarantee recoverability, the
-:tool:`condor_dagman` process is run as an HTCondor job. As such, it needs a
-submit description file. :tool:`condor_submit_dag` generates this needed
-submit description file, naming it by appending ``.condor.sub`` to the
-name of the DAG input file. This submit description file may be edited
-if the DAG is submitted with
+To increase robustness and guarantee recoverability, the :tool:`condor_dagman`
+process is run as an HTCondor job. As such, it needs a submit description file.
```

Comment:
I think we should say something about "process is run as a HTCondor Scheduler universe jobs that runs under the schedd on the AP".  This avoids confusion that some might think it is a vanilla job running on some EP.